glib/src/auto/functions.rs

```diff
@@ -249,6 +250,17 @@ pub fn file_set_contents_full(
     }
 }
 
+#[doc(alias = "g_file_test")]
+#[allow(dead_code)]
+pub(crate) fn file_test(filename: impl AsRef<std::path::Path>, test: FileTest) -> bool {
```

Comment:
Isn't this proper public now via the `pub use auto::functions::*` in `lib.rs`?

Reply:
Not really, hence the allow dead_code but I will double check

Reply:
Hm I thought that's how you'd change the visibility of items to something else, by re-exporting from a place where it is accessible :)